src/Chapter04/Listing04.45.ForLoop.cs

```diff
@@ -11,7 +11,9 @@ public static void Main()
             System.Console.Write("Enter an integer: ");
             // Use long.Parse() to support negative numbers
             // Assumes unchecked assignment to ulong
-            value = (ulong)long.Parse(System.Console.ReadLine());
+            // TODO: Update listing in Manuscript
+            // If ReadLine returns null, use "42" as default input
+            value = (ulong)long.Parse(System.Console.ReadLine() ?? "42");
```

Comment:
Since the sample output in the book for this one shows "-42" I would propose we use it for the default.
```suggestion
            value = (ulong)long.Parse(System.Console.ReadLine() ?? "-42");
```